core/vm/evm.go

```diff
@@ -439,6 +439,15 @@ func (evm *EVM) create(caller ContractRef, codeAndHash *codeAndHash, gas uint64,
 	if evm.StateDB.GetNonce(address) != 0 || (contractHash != (common.Hash{}) && contractHash != types.EmptyCodeHash) {
 		return nil, common.Address{}, 0, ErrContractAddressCollision
 	}
+
```

Comment:
We're already changing where the hooks are called in #27629. Honestly I'd rather do it once and publish a semantic-change notice to users.

Reply:
I understand that this may affect the semantics of the current implementation, and that's fine to not merge this one.